internal/rpchelp/helpdescs_en_US.go

```diff
@@ -707,6 +707,7 @@ var helpDescsEnUS = map[string]string{
 	"purchaseticket-nosplittransaction": "Use ticket purchase change outputs instead of a split transaction",
 	"purchaseticket-comment":            "Unused",
 	"purchaseticket-ticketfee":          "The transaction fee rate (DCR/kB) to use (overrides fees set by the wallet config or settxfee RPC)",
+	"purchaseticket-dontsigntx":         "When true creates unsigned tickets tx and return its bytes and its split tx (default=\"false\")",
```

Comment:
I'd suggest this wording, which is more brief and more closely matches the written style of the other help messages:

> Return unsigned split and ticket transactions instead of signing and publishing